test/integration/proxy_test.js

```diff
@@ -715,7 +715,11 @@ test('reroutes retry to remote', function t(assert) {
             retrySchedule: [1]
         }, function onRequest(err, res) {
             assert.ifError(err, 'no error occurs');
-            assert.equal(res.body, 'rerouted', 'response from rerouted request is correct');
+            assert.equal(
+                res.body.toString('hex'),
+                new Buffer('rerouted').toString('hex'),
+                'response from rerouted request is correct'
```

Comment:
If my understanding of the backwards-compatibility fix (604350879c41b823eb69eb04a1a7382d8ae961c3) is correct, we no longer need to change the tests, right? Body will be string at this point?